pkg/server/webhook.go

```diff
@@ -531,6 +531,14 @@ func (whsvr *WebhookServer) mutate(req *v1beta1.AdmissionRequest) *v1beta1.Admis
 		}
 	}
 
+	if injectionConfig.SkipHostNetwork && pod.Spec.HostNetwork {
+		glog.Infof("Injection skipped by SkipHostNetwork")
```

Comment:
[nit] maybe a more expressive message like "injection skipped due to pod using HostNetwork, and SkipHostNetwork is enabled for this sidecar"?

Reply:
Thanks for your review! I've pushed a new commit which prints readable skipping reason, as well as the ns/name of the pod and the name of the sidecar config.